interpreter.js

```diff
@@ -2773,22 +2773,22 @@ Interpreter.prototype['stepCallExpression'] = function() {
   if (state.doneCallee_ === 1) {
     // Determine value of the function.
     state.doneCallee_ = 2;
-    var func = state.value;
-    if (Array.isArray(func)) {
-      state.func_ = this.getValue(func);
-      state.components_ = func;
-      func = state.func_;
-      if (typeof func === 'object' && func.isGetter) {
+    var stateValue = state.value;
```

Comment:
is there a specific reason to rename this variable?

Reply:
I can change it back, but func was being used and re-used in so many places that it was turning into a bit of a headache for me. I figured I'd clean it up a bit for the next person. Also, I would argue that the overuse of this variable name is part of the reason this bug exists.

Reply:
ah, I can see that. I have no objection to changing it necessarily, but I would recommend finding something more descriptive than `var stateValue = state.value;`. The benefit of the original is that it communicates the message that `state.value` is expected to represent a function, which this approach does not.

Reply:
replaced with better naming.

Reply:
removed this section of changes to keep this as surgical as possible.